tests/core/netplan-cfg/task.yaml

```diff
@@ -2,7 +2,9 @@ summary: Ensure that netplan config integration works
 
 # TODO: enable for all ubuntu core versions once netplan.io got SRUed
 #       into the older versions of ubuntu
-systems: [ubuntu-core-20-*]
```

Comment:
You could use
systems: [ubuntu-core-2*]

Reply:
It is fine to use globs to match all archs, all distros, or all versions. But for a subset of versions no. Because it is just a coincidence that we use base 10 to write the version and that this test apply from version 20. But we were to match something for ubuntu-core-28 and ubuntu-core-30 for example, we would not use a glob. And we would have to use a list then. For that reason it is better for consistency to always list. We can always use a list.

It would be nice however if spread could have notion of range on some components on the system name.